go.mod

```diff
@@ -4,12 +4,12 @@ go 1.18
 
 require (
 	github.com/cloudflare/cfssl v1.6.1
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
 	k8s.io/klog/v2 v2.70.1
-	open-cluster-management.io/addon-framework v0.4.0
-	open-cluster-management.io/api v0.8.0
+	open-cluster-management.io/addon-framework v0.5.0
```

Comment:
we fixed some issues in the famework recently, please use the latest addon-framework

```
go get -v -u open-cluster-management.io/addon-framework@main
```

Reply:
Done